StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -368,8 +368,16 @@ class AddPaymentMethodViewController: UIViewController {
                 from: viewController,
                 financialConnectionsCompletion: financialConnectionsCompletion
             )
-        case .deferredIntent:
-            fatalError("TODO(DeferredIntent): Support ACHv2")
+        case let .deferredIntent(elementsSession, intentConfig):
+            // I observed a blank webview unless the session ID is unique per attempt, so append a UUID.
+            let sessionId = elementsSession.sessionID + "--" + UUID().uuidString
```

Comment:
cc @jameswoo-stripe

Reply:
Hmm this is interesting, does web need to do anything like this?

Reply:
No, got to the bottom of this here https://stripe.slack.com/archives/C04HHL5CN1G/p1682717486369159 - ACHv2 decoupled support is blocked until we fix it